### PR TITLE
Updated the spreadhseet to v1.1.0 from 0.9.0. vbproj and package.config added

### PR DESCRIPTION
**Pull request by deaspo**

### CODE REVIEW

**Comment by deaspo** [comment]
Hey @dannyparsons , I think this fix the issue addressed earlier of vbproj

File: instat/instat.vbproj [hunk 1]
```diff
@@ -116,9 +106,9 @@
     <Reference Include="System.Xml.Linq" />
     <Reference Include="System.Data.DataSetExtensions" />
     <Reference Include="System.Net.Http" />
-    <Reference Include="unvell.ReoGrid, Version=0.8.5.0, Culture=neutral, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>References\unvell.ReoGrid.dll</HintPath>
+    <Reference Include="unvell.ReoGrid, Version=1.1.0.0, Culture=neutral, PublicKeyToken=0a947d9ccc179105, processorArchitecture=MSIL">
+      <HintPath>..\packages\unvell.ReoGrid.dll.1.1.0.0\lib\20\unvell.ReoGrid.dll</HintPath>
```
Comment (dannyparsons) [inline instat/instat.vbproj]: Shouldn't this be in the References still? As it is in line 121 in previous version?

File: instat/instat.vbproj [hunk 2]
```diff
@@ -83,17 +83,10 @@
...
-    </Reference>
     <Reference Include="DynamicInterop, Version=0.7.4.0, Culture=neutral, processorArchitecture=MSIL">
       <HintPath>..\packages\DynamicInterop.0.7.4\lib\net40\DynamicInterop.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="FastColoredTextBox">
-      <HintPath>References\FastColoredTextBox.dll</HintPath>
-    </Reference>
     <Reference Include="RDotNet, Version=1.6.5.0, Culture=neutral, processorArchitecture=MSIL">
       <HintPath>..\packages\R.NET.Community.1.6.5\lib\net40\RDotNet.dll</HintPath>
```
Comment (deaspo) [inline instat/instat.vbproj]: @dannyparsons  looking at line 98, `<HintPath>`, I think the hint paths are different when you import a Reference and when you install a reference from the .Nuget package manager.

File: instat/instat.vbproj (hunk 1 above)
Comment (deaspo) [inline instat/instat.vbproj]: I think the first time I added the grid was through right clicking the reference>add reference>browse. But this time I did install from the Nuget package manager. I think that's what is bringing the difference

**Comment by dannyparsons** [comment]
Thanks for noticing the different uses of packages and reference. I looked into it in detail with @volloholic and although both ways work, we think we should be doing everything through References and not Packages. If we reference, then we can have all packages in a 'lib' folder in our project. Then anyone getting the project from GitHub has all the packages downloaded and referenced correctly already. An answer here explains it well:

> As most other answers have hinted, you can just add a reference to a dll from anywhere. However it'd probably be a good idea to create a 'lib' (as in library) folder in your project, copy the dll in there and only then reference it. That way if you share your solution via source control then anyone else can use it too. (without having to download or add their own reference to a copy of the dll)
http://stackoverflow.com/questions/1329196/where-should-i-place-the-dll

Could you and @maxwellfundi work on this to get all the packages we need in a 'lib' folder in the project and referenced properly?

**Comment by dannyparsons** [comment]
Also, since the 1.2 version of the grid fixes the radio button issue we were having, we think it's worth using 1.2 even though it's not the release version yet.
You were right to stick with the release version, we shouldn't release Instat with a developer version, but it looks like this will be a release version by the time we get to Beta.

**Comment by deaspo** [comment]
Sure, currently doing that.

**Comment by deaspo** [comment]
But first I have to close this pull request, okay?

**Pull request closed without merging**